### ISSUE TITLE
tempo query can't show service name

### ISSUE THREAD

**Issue by hellertang** [grafana/tempo#578]
**Describe the bug**
i can search trace by trace id , but can't find service name show in fronted.

![image](https://user-images.githubusercontent.com/10046299/110432952-cf169b80-80ea-11eb-8ece-1ca7f39375c2.png)

![image](https://user-images.githubusercontent.com/10046299/110432962-d178f580-80ea-11eb-9257-e505620ee92c.png)

**Client**
base on opentelemetry , use zipkin exporter, push to port 9411
```
from opentelemetry import trace as trace_api
from opentelemetry.sdk.trace import TracerProvider
from opentelemetry.sdk.trace.export import BatchExportSpanProcessor

from opentelemetry.exporter import zipkin

zipkin_exporter = zipkin.ZipkinSpanExporter(
    service_name="Django_jaeger——test",
    url="http://xxxxxx:9411/api/v2/spans"
)

trace_api.set_tracer_provider(TracerProvider())
tracer = trace_api.get_tracer_provider().get_tracer(__name__)

trace_api.get_tracer_provider().add_span_processor(
    BatchExportSpanProcessor(zipkin_exporter),
)

with tracer.start_as_current_span("foo"):
    parent_span_context = trace_api.get_current_span().get_span_context()
    trace_id = parent_span_context.trace_id
    print(hex(trace_id))
    print("Hello world from OpenTelemetry Python!")
```
the same client , i push to jaeger-collector, query by jaeger-query, it works ok~

![image](https://user-images.githubusercontent.com/10046299/110433465-71368380-80eb-11eb-9885-87fa8f106b40.png)

is there anyone encountered the same error ? help~

**Comment by dgzlopes** [comment]
Hi @hellertang ! Thanks for creating the ticket :) 

This is the expected behavior. 

Right now, Tempo can only retrieve traces given a traceID and that's why the drop-down doesn't list anything.

**Comment by hellertang** [comment]
i may know why it could be difficult for tempo to support query by service_name. 
Tempo store in s3,  Jaeger collector store in es, different data structures~

Will tempo plan to support this in future? not everyone use it integrated with loki, i amazed tempo because it support store in s3

**Comment by dgzlopes** [comment]
Yup, there are plans to support this in the future, but still, it's too early to know how this would look like.

btw, keep in mind that if you don't want to use Loki... You don't have to! You can use any logging backend to perform the `trace_id` discovery step. [Here](https://grafana.com/blog/2020/12/07/how-to-find-traces-in-tempo-with-elasticsearch-and-grafana/) we have an example with Elasticsearch.

Also, if you use Prometheus, Exemplars support is coming. And that means that you [can discover traces](https://grafana.com/blog/2020/12/07/how-to-find-traces-in-tempo-with-elasticsearch-and-grafana/), from the metrics!

**Comment by hellertang** [comment]
> Yup, there are plans to support this in the future (or at least conversations around this idea), but still, it's too early to know how this would look like.
> 
> btw, keep in mind that if you don't want to use Loki... You don't have to! You can use any logging backend to perform the `trace_id` discovery step. [Here](https://grafana.com/blog/2020/12/07/how-to-find-traces-in-tempo-with-elasticsearch-and-grafana/) we have an example with Elasticsearch [0].
> 
> Also, if you use Prometheus, Exemplars support is coming. And that means that you [can discover traces](https://grafana.com/blog/2020/11/09/trace-discovery-in-grafana-tempo-using-prometheus-exemplars-loki-2.0-queries-and-more/), from the metrics! [1]
> 
> [0] https://grafana.com/blog/2020/12/07/how-to-find-traces-in-tempo-with-elasticsearch-and-grafana/
> [1] https://grafana.com/blog/2020/11/09/trace-discovery-in-grafana-tempo-using-prometheus-exemplars-loki-2.0-queries-and-more/
wow, 3Q~ 
s3 is cheaper than es:)
